main.py

```diff
@@ -0,0 +1,113 @@
+import sqlite3
+import random
+import threading
+import os
+import dbActions
```

Comment:
[Modules should have short, all-lowercase names.](https://www.python.org/dev/peps/pep-0008/#package-and-module-names)